src/Dijkstra/AST/Branching.agda

```diff
@@ -289,6 +282,14 @@ module BranchingSyntax (BaseOps : ASTOps) where
                              ; (lift (just a)) → fA a
                              }
 
+  instance
+    open import Dijkstra.AST.Syntax
+    open import Dijkstra.Syntax
+    open MonadMaybeD public
```

Comment:
Speaking of "MaybeD" - what is the status of "MaybeD"?

A search shows it is used in PeerHandlerContracts and BlockTree-AST.

I am thinking, in an ideal world, this "old" stuff should go into a separate file (or be deprecated/replaced entirely).

Reply:
There may be some confusion here.  MaybeD :-) you searched on the wrong branch?  Or maybeD you're talking about `MonadMaybeD`?  The latter is only about syntax, enabling the use of `maybeSD`, `caseMD`, etc.  It's nothing to do with the differences between the "old" and "new" style of reasoning, and does not need to be updated or eliminated.